From ea139a4b188845f27af2448e2044461bedfae90a Mon Sep 17 00:00:00 2001
From: Mei Liang <meiliang@chromium.org>
Date: Thu, 08 Jun 2023 07:35:07 +0000
Subject: [PATCH] [114][ChromeCart][Quest] Fix the in your card annotation width

We spotted this on Mac. Updating the width attribute to min-width fix
the bug.

Before change: https://screenshot.googleplex.com/7ZVnjAXpGUXNtGz
After change: https://screenshot.googleplex.com/BoT3J7WHJYjdDTR

(cherry picked from commit 3d6d6e95f25a1ca8fc4bdd51d5019690eb0c8b2a)

Bug: 1449035 b/284206972
Change-Id: I7178d7939db060370097da53b1607ece473be5c7
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4575453
Commit-Queue: Mei Liang <meiliang@chromium.org>
Reviewed-by: Tibor Goldschwendt <tiborg@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1150991}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4599039
Cr-Commit-Position: refs/branch-heads/5735@{#1199}
Cr-Branched-From: 2f562e4ddbaf79a3f3cb338b4d1bd4398d49eb67-refs/heads/main@{#1135570}
---

--- a/chrome/browser/resources/new_tab_page/modules/history_clusters/cart/cart_tile.html
+++ b/chrome/browser/resources/new_tab_page/modules/history_clusters/cart/cart_tile.html
@@ -121,7 +121,7 @@
   #titleAnnotation {
     background: var(--color-sys-tonal-container);
     color: var(--color-sys-on-tonal-container);
-    width: 60px;
+    min-width: 60px;
   }
 
   :host([one-image]) #titleAnnotation {
